### ISSUE TITLE
Multiline editing broken

### ISSUE THREAD

**Issue by davidbrochart** [ipython/ipython#11337]
In IPython 7.0.1, it is only possible to enter e.g. a one-line function:

```python
def foo():
    print('Hello')
    # cell is entered automatically here
```

**Comment by Carreau** [comment]
Duplicate of #11334, you can use `ctrl-o` to force a new line.

**Comment by Carreau** [comment]
Actually it might not be a duplicate, it may be more complicated than that.

**Comment by Carreau** [comment]
closed by #11358
